gunicorn/glogging.py

```diff
@@ -199,7 +204,11 @@ def setup(self, cfg):
                 self.access_log, cfg, self.syslog_fmt, "access"
             )
 
-        if cfg.logconfig:
+        if dictConfig and cfg.logconfig_dict:
```

Comment:
I think we can print a user friendly message if `dictConfig is None` instead of silently ignore the problem. I will review this more carefully soon. Sorry for my late response!